src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+
+  switch (true) {
+    case days >= 3 && days < 7:
+      return days * rentPrice - minDiscount;
+    case days >= 7:
+      return days * rentPrice - maxDiscount;
+    default :
+      return days * rentPrice;
+  }
```

Comment:
don't use switch case with `true` it's a wrong way how to use switch case, in this case it's better to use a few `if`

Reply:
fixed